lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -115,6 +115,7 @@ def _parse_dist_file(self, name, dist_file_content, path, collected_facts):
             if self.SEARCH_STRING[name] in dist_file_content:
                 # this sets distribution=RedHat if 'Red Hat' shows up in data
                 dist_file_dict['distribution'] = name
+                dist_file_dict['distribution_file_search_string'] = self.SEARCH_STRING[name]
```

Comment:
What's the motivation for this fact? Is it used anywhere or otherwise requested?

Reply:
Mostly for troubleshooting issues like this.